### ISSUE TITLE
Revisited: Missing multiclass-multioutput support

### ISSUE THREAD

**Issue by tron27** [automl/auto-sklearn#1685]
Good morning all, 
I am revisiting the concern regarding if/when auto-sklearn will provide support for multiclass-multioutput data. The original issue can be found below: 

https://github.com/automl/auto-sklearn/issues/292

In brief, it was mentioned (from the link above) that once scikit-learn provides metrics to evaluate multiclass-multioutput predictions, there will be a clean issue for auto-sklearn to also provide support for multiclass-multioutput data. It seems as though scikit-learn now offers/supports multiclass-multioutput. See the link below:

https://scikit-learn.org/stable/modules/multiclass.html

My question is, does any of the contributors know where auto-sklearn is regarding support for this feature? I would like to apply automl to predict a location (latitude/longitude). Lastly, I would like to thank all of the contributors for all that you do! I and the community greatly appreciate it. Attached is a screenshot of me trying to fit my X_train and y_train data. Unfortunately, I get an error regarding continuous-multioutput not being supported (see screenshot below). As you can see, I'm trying to apply automl in four lines of code. 

Continuous multioutput error:
<img width="1390" alt="continuous multioutput error" src="https://github.com/automl/auto-sklearn/assets/12566235/5adfbc97-4fa7-4781-9e00-4a69cad9f996">

System Details:
Google Colab which runs on Ubuntu version 22.04 (see screenshot)
<img width="1402" alt="Google Colab OS" src="https://github.com/automl/auto-sklearn/assets/12566235/6ddda754-1d67-420b-bdf6-be153acf18e0">

auto-sklearn: 0.16.0.dev0 (see screenshot) - Thank you to @Frankothe196 for helping me with this!
<img width="1374" alt="auto-sklearn install" src="https://github.com/automl/auto-sklearn/assets/12566235/64c52f3f-543d-4f60-8b50-4092fe1a1063">

<img width="1375" alt="auto-sklearn version" src="https://github.com/automl/auto-sklearn/assets/12566235/b606f6b2-8c1f-4a22-911b-c5342edf415b">

Thanks,
Eric

# System Details (if relevant)
* Which version of `auto-sklearn` are you using?
* Are you running this on Linux / Mac / ... ?

**Comment by charlesfu4** [comment]
Hi @tron27. If I understood it correctly. You're trying to conduct a classification task with labels in continuous variables type which is not possible. Have you tried making the continuous variable binned or into categorial ones?
If the purpose is to predict continuous variables, instead of using classification models, one should go with regression models.

Kr,
Charles
